src/xcode/ENA/ENA/Source/HTTPClient/Resources/ServiceIdentityDocumentResource.swift

```diff
@@ -11,6 +11,7 @@ enum ServiceIdentityDocumentResourceError: Error {
 	case VS_ID_SERVER_ERR
 	case VS_ID_CERT_PIN_NO_JWK_FOR_KID
 	case VS_ID_CERT_PIN_MISMATCH
```

Comment:
Can these be removed or can they still occur?

Reply:
we still need the RTR_CERT_PIN_MISMATCH for one of the allowlist errors, I removed the other